content/en/Reference/aws/modules/aws-mysql.md

```diff
@@ -17,6 +17,14 @@ Opta will provision your database with 7 days of automatic daily backups in the
 You can find them either programmatically via the aws cli, or through the AWS web console (they will be called
 system snapshots, and they have a different tab than the manual ones).
 
+### Performance and Scaling
+
+You can modify the DB instance class with the field `instance_class` in the module configuration.
```

Comment:
Can we add a link to AWS documentation about DB instance classes available for mysql here?